.drone.yml

```diff
@@ -1,54 +1,209 @@
+---
 kind: pipeline
 type: docker
-name: mrgsolve
+name: mpn:latest
+
+platform:
+  os: linux
+  arch: amd64
 
 steps:
-- name: Pull mpn container from ECR
+- name: pull
   image: omerxx/drone-ecr-auth
+  commands:
+  - $(aws ecr get-login --no-include-email --region us-east-1)
+  - docker pull 906087756158.dkr.ecr.us-east-1.amazonaws.com/r-dev-ci-mpn-4.1:latest
+  - docker pull 906087756158.dkr.ecr.us-east-1.amazonaws.com/r-dev-ci-mpn-4.0:latest
+  - docker pull 906087756158.dkr.ecr.us-east-1.amazonaws.com/r-dev-ci-mpn-3.6:latest
```

Comment:
You can delete this line because you don't actually use this container.

As discussed offline, the `r-dev-ci-mpn-3.6:latest` container hasn't been built since May 2021 so it doesn't have the required versions of deps. I think it's fine to drop it from here completely, especially since you're testing 3.6 against CRAN-latest below.